src/platforms/react-native/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -170,6 +176,46 @@ new Sentry.ReactNavigationV4Instrumentation({ ... });
 
 How long the instrumentation will wait for the **initial** route to mount after a change has been initiated before the transaction is discarded. Default: 1000ms
 
+#### React Native Navigation
+
+<Note>
+
+Our React Native Navigation instrumentation is currently in beta and is available from version `3.2.0-beta.1` onwards.
+
+</Note>
+
+This instrumentation supports [react-native-navigation](https://github.com/wix/react-native-navigation). You will need to pass the `Navigation` object imported from the library to initialize our routing instrumentation. It is recommended that you initialize our SDK and the routing instrumentation as early as possible in the lifecycle of your app, this most likely would be before where you initialize your screens.
```

Comment:
```suggestion
This instrumentation supports [react-native-navigation](https://github.com/wix/react-native-navigation). You will need to pass the `Navigation` object imported from the library to initialize our routing instrumentation. It is recommended that you initialize our SDK and the routing instrumentation as early as possible in the lifecycle of your app; this would most likely be before the point where you initialize your screens.
```